enterprise/ha/src/main/java/org/neo4j/kernel/ha/transaction/OnDiskLastTxIdGetter.java

```diff
@@ -33,11 +33,15 @@ public OnDiskLastTxIdGetter( Supplier<NeoStores> neoStoresSupplier )
         this.neoStoresSupplier = neoStoresSupplier;
     }
 
+    // This method is used to construct credentials for election process.
+    // And can be invoked at any moment of instance lifecycle.
+    // It mean that its possible that we will be invoked when neo stores are stopped
+    // (for example while we copy store)
     @Override
     public long getLastTxId()
     {
         TransactionIdStore neoStore = getNeoStores().getMetaDataStore();
-        return neoStore.getLastCommittedTransactionId();
+        return neoStore != null ? neoStore.getLastCommittedTransactionId() : TransactionIdStore.BASE_TX_ID;
```

Comment:
It's not null - it's closed like in test - we calling `neoStores.close`as result of that after that call calling `getMetaDataStore()` will return null since neoStores are closed.